### PR TITLE
FIX - Audit fixes

### PR DESCRIPTION
**Pull request by halaprix**
- fix `5.21 Unused Parameter, Variable and Event`

### CODE REVIEW

File: contracts/adapters/AAVEAdapter.sol [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
 import "../interfaces/ManagerLike.sol";
 import "../interfaces/BotLike.sol";
 import "../interfaces/MPALike.sol";
+import "../interfaces/IAdapter.sol";
 import "../ServiceRegistry.sol";
 import "../McdView.sol";
 import "../McdUtils.sol";
 
-contract AAVEAdapter {
+contract AAVEAdapter is IAdapter {
```
Comment (adamskrodzki) [inline contracts/adapters/AAVEAdapter.sol:31]: https://github.com/OasisDEX/automation-smartcontracts/pull/161

proposal of different approach

File: contracts/adapters/AAVEAdapter.sol [hunk 2]
```diff
@@ -44,8 +61,12 @@ contract AAVEAdapter {
                 amount
             )
         );
-
-        //todo: call proxy to withdraw coverageToken from aave
-        //todo: call proxy to transfer coverageToken to receiver
     }
+
+    function canCall(bytes memory, address) external virtual returns (bool) {}
```
Comment (adamskrodzki) [inline contracts/adapters/AAVEAdapter.sol:66]: remove it or at best revert inside if code is not functional

File: contracts/adapters/DPMAdapter.sol [hunk 3]
```diff
@@ -22,11 +22,12 @@ import "../interfaces/IAccountGuard.sol";
 import "../interfaces/ManagerLike.sol";
 import "../interfaces/BotLike.sol";
 import "../interfaces/MPALike.sol";
+import "../interfaces/IAdapter.sol";
 import "../ServiceRegistry.sol";
 import "../McdView.sol";
 import "../McdUtils.sol";
 
-contract DPMAdapter {
+contract DPMAdapter is IAdapter {
```
Comment (adamskrodzki) [inline contracts/adapters/DPMAdapter.sol:30]: https://github.com/OasisDEX/automation-smartcontracts/pull/161

File: contracts/adapters/MakerAdapter.sol [hunk 4]
```diff
@@ -21,11 +21,12 @@ import "../interfaces/ICommand.sol";
 import "../interfaces/ManagerLike.sol";
 import "../interfaces/BotLike.sol";
 import "../interfaces/MPALike.sol";
+import "../interfaces/IAdapter.sol";
 import "../ServiceRegistry.sol";
 import "../McdView.sol";
 import "../McdUtils.sol";
 
-contract MakerAdapter {
+contract MakerAdapter is IAdapter {
```
Comment (adamskrodzki) [inline contracts/adapters/MakerAdapter.sol:29]: https://github.com/OasisDEX/automation-smartcontracts/pull/161

File: test/automation-bot.ts [hunk 5]
```diff
@@ -204,6 +198,27 @@ describe('AutomationBot', async () => {
             const receipt = await tx.wait()
             const events = getEvents(receipt, AutomationBotInstance.interface.getEvent('TriggerAdded'))
             expect(events.length).to.be.equal(1)
+            expect(events[0].args.triggerId).to.be.equal(1)
+        })
+
+        it.only('should emit TriggerAdded if called by user being an owner of proxy and the id[0] is == 1', async () => {
```
Comment (adamskrodzki) [inline test/automation-bot.ts:204]: remove only

**Review by adamskrodzki — CHANGES_REQUESTED** [review]

File: contracts/AutomationBot.sol [hunk 6]
```diff
@@ -353,8 +358,6 @@ contract AutomationBot {
 
     event ApprovalRemoved(bytes indexed triggerData, address approvedEntity);
 
-    event ApprovalGranted(bytes indexed triggerData, address approvedEntity);
```
[the remark this answers is not in the document]
Comment (halaprix) [reply contracts/AutomationBot.sol:356]: added

File: test/automation-bot.ts (hunk 5 above)
Comment (halaprix) [reply test/automation-bot.ts:204]: done

**Review by adamskrodzki — APPROVED** [review]

**Pull request merged**